src/vcfio/vcf_writer.py

```diff
@@ -41,10 +41,23 @@ def write_headers(self, headers: Iterable[AnyStr] = ()):
         if not headers:
             headers = [REQUIRED_HEADERS[0]]
             self._write_last_header = True
-        for header in headers:
+        sorted_headers = self.sort_headers(headers)
+        for header in sorted_headers:
             self._file_descriptor.write(header)
             self._file_descriptor.write('\n')
 
+    def sort_headers(self, headers: Iterable[AnyStr] = ()):
+        def custom_sort(line):
```

Comment:
lets add a condition for line that is not a header. if there is a logging - lets add a warning